examples/package.json

```diff
@@ -9,10 +9,11 @@
   },
   "author": "Lars Moastuen <lars.moastuen@cognite.com>",
   "contributors": [
-    "Lars Moastuen <lars.moastuen@cognite.com>",
-    "Joachim Bjørne <joachim.bjorne@cognite.com>",
     "Christopher Tannum <christopher.tannum@cognite.com>",
-    "Maksim Nesterenko <maksim.nesterenko@cognite.com>"
```

Comment:
Should we keep old contributors as well? Just asking, not sure what the convention is :sweat_smile: